cpp/src/arrow/acero/swiss_join_avx2.cc

```diff
@@ -194,17 +202,19 @@ int RowArrayAccessor::Visit_avx2(const RowTableImpl& rows, int column_id, int nu
       //
       const uint8_t* row_ptr_base = rows.data(2);
       const RowTableImpl::offset_type* row_offsets = rows.offsets();
+      auto row_offsets_i64 =
+          reinterpret_cast<const arrow::util::int64_for_gather_t*>(row_offsets);
       for (int i = 0; i < num_rows / unroll; ++i) {
```

Comment:
Can we take the opportunity to rename all these `unroll` constants to `kUnroll`?

Reply:
Done.